### ISSUE TITLE
clean npm install of vue-styleguidist fails

### ISSUE THREAD

**Issue by pop42** [vue-styleguidist/vue-styleguidist#1296]
**Current behavior**

At this time, trying to install vue-styleguidist will cause npm install to go into a constant loop trying to install sub-dependencies react, react-dom and react-codemirror2 presumably from react-dev-tools as a dependency in vue-styleguidist

**To reproduce**

The following repo is a basic npm init with only one package to install.. vue-styleguidist, but npm will fail to install it.
https://github.com/pop42/vue-styleguidist-issue

➜  vue-styleguidist-issue git:(main) npm --version
7.21.1
➜  vue-styleguidist-issue git:(main) node --version
v16.9.1

**Expected behavior**

I expect to be able to run `npm i -D vue-styleguidist` to install vue-styleguidist.

**Comment by elevatebart** [comment]
I do not see this issue with npm 8.3.1. (node 16.14.0)
Can you upgrade npm and see what happens?

**Comment by elevatebart** [comment]
I also see that you are using IntelliJ, it might be linked to the intelligence that comes with it.

**Comment by elevatebart** [comment]
Also, `gitignore` should contain at least `node_modules`

I hope this helps.

**Comment by pop42** [comment]
I discovered the issue outside of the project using intellij and with a proper .gitignore, this was just a quick project I created to be user the package.json only included vue-styleguidist for this ticket.  

But... you are 100% correct it was related to the version of npm. I upgrated npm `npm i -g npm@latest`  -> 8.5.5     

Then an install works of vue-styleguidist. 

I should have attempted that first, but it didn't occur to me.  You are awesome!

Thanks!
